### PR TITLE
MPD. Update frontend installation instructions. Reduce redundancy.

### PR DESCRIPTION
**Pull request by ksuess**
MPD has two chapters with frontend installation instructions.
grafik

This PR reduces redundancy and therefore maintenance by
- refering in first chapter to Plone 6 docs.
- refering in second chapter to first chapter.

### CODE REVIEW

**Comment by ksuess** [comment]
> 😎 Deploy Preview 	https://deploy-preview-621--plone-training.netlify.app
> 

Reviewing is easy, @pbauer

File: docs/mastering-plone/installation.md [hunk 1]
```diff
@@ -99,79 +99,115 @@ For a Plone 6 installation by now two installations are needed: Plone backend an
...
+yarn start
+```
 
-1. Generate a project with {term}`Yeoman` and the Volto generator:
+This will start the server on port 3000.
 
-   ```shell
-   yo @plone/volto
-   ```
+You can change the port and/or hostname for the frontend by specifying PORT and/or HOST:
```
Comment (stevepiercy) [inline docs/mastering-plone/installation.md:195]:
```suggestion
You can change the port and/or hostname for the frontend by specifying `PORT` and/or `HOST`:
```

File: docs/mastering-plone/installation.md [hunk 2]
```diff
@@ -99,79 +99,115 @@ For a Plone 6 installation by now two installations are needed: Plone backend an
...
+```
 
-If successful, you get:
 
-```console
-🎭 Volto started at <http://localhost:3000> 🚀
-```
+## Creating a Plone Website
 
+Now, that you have a backend and a frontend up and running, you can create your concrete website for a project.
```
Comment (stevepiercy) [inline docs/mastering-plone/installation.md:210]:
```suggestion
Now that you have a backend and a frontend up and running, you can create your concrete website for a project.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor fixes. Otherwise, LGTM.

**Pull request merged**